src/test/java/org/folio/rest/api/LoanPoliciesApiTest.java

```diff
@@ -190,6 +191,19 @@ public void loanPolicyWithFixedDate() throws
     TextResponse updateResponse = updateCompleted.get(5, TimeUnit.SECONDS);
     assertThat("update loanPolicy: set alternateFixedDueDateScheduleId = non existent id",
         updateResponse, matchesBadRequest());
+
+    // loanable==false but loansPolicy exist
+    JsonObject loanPolicyRequest9 = new JsonObject()
+        .put("name", "9").put("loanable", false).put("renewable", false)
+        .put("loansPolicy", new JsonObject());
+    CompletableFuture<TextResponse> completed9 = new CompletableFuture<>();
+    client.post(loanPolicyStorageUrl(),
+      loanPolicyRequest9, StorageTestSuite.TENANT_ID,
+      ResponseHandler.text(completed9));
+    TextResponse response9 = completed9.get(5, TimeUnit.SECONDS);
+    assertThat("loanable==false but loansPolicy exist", response9, matchesUnprocessableEntity());
+    // TODO: consider returning a better message
+    assertThat(response9.getBody(), containsString("Fixed due date"));
```

Comment:
Please could this be defined as a separate test method rather than hidden amongst these other tests in the same method.

I think it would be useful, given that this code is changing to have a test demonstrating each of the four validations.

Reply:
cannotUpdateWhenFixedIsRenewable() and loanPolicyWithFixedDate() check the 1st validation.
cannotCreateLoanPolicyWithLoanableFalseAndLoansPolicy() and loanPolicyWithFixedDate() check the 2nd validation.
cannotUseHoldAlternateRenewalLoanPeriodForFixedProfile() checks the 3rd validation.
cannotUseRenewalsPeriodForFixedProfile() checks the 4th validation.

Sonar reports that all four rules are covered by tests:
https://sonarcloud.io/component_measures?id=org.folio%3Amod-circulation-storage&metric=new_coverage&pullRequest=250&selected=org.folio%3Amod-circulation-storage%3Asrc%2Fmain%2Fjava%2Forg%2Ffolio%2Frest%2Fimpl%2FLoanPoliciesAPI.java&view=list